main-project/src/boards/dto/update-board.dto.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  IsBoolean,
+  IsDate,
+  IsNotEmpty,
+  IsNumber,
+  IsString,
+} from 'class-validator';
+
+export class UpdateBoardDto {
+  @IsNotEmpty()
+  @IsString()
+  title: string;
+
+  @IsNotEmpty()
+  @IsString()
+  description: string;
+
+  @IsBoolean()
+  @IsNotEmpty()
+  done: boolean;
+
+  @IsString()
+  location: string;
+
+  @IsDate()
+  time: Date;
+
+  @IsNotEmpty()
+  @IsNumber()
+  male: number;
+
+  @IsNotEmpty()
+  @IsNumber()
+  female: number;
```

Comment:
```suggestion
  @IsNotEmpty()
  @IsBoolean()
  isGender: boolean;
```
genderYN도 가능하긴 합니다.
성별의 경우 남자, 여자 둘 중 하나인 경우니 boolean값으로 관리해도 좋습니다.

Reply:
앗 이 필드들은 남성 몇 명, 여성 몇 명 인지 인원설정하는 필드입니다!